addons/tagging/ACE_Tags.hpp

```diff
@@ -1,3 +1,10 @@
+#define TAG(name,col) class TRIPLES(ACE,name,col) { \
+        displayName = CSTRING(DOUBLES(name,col)); \
+        requiredItem = QUOTE(ACE_Spraypaint##col); \
```

Comment:
```suggestion        
        requiredItem = QUOTE(GLUE(ACE_Spraypaint,col)); \
```

Reply:
fixes 
`[ACE] (tagging) ERROR: Failed compiling ACE_Tags for tag: ACE_ar_u_Black - requiredItem ACE_Spraypaint##Black does not exist`